web/style/products.scss

```diff
@@ -0,0 +1,130 @@
+@import '_variables.scss';
+@import 'common.scss';
+.category-list {
+    width: 25%;
+    padding: 3px 12px 3px 12px;
+    background-color: #eaeaea;
+    margin-top: -10px;
+    display: none;
+  
+    @include break-point(tablet) {
+      display: block;
+    }
+  
+    &__name {
+      font-size: 1rem;
+      padding: 12px 0 12px 10px;
+      color: #959595;
+      border-bottom: 1px solid #cecccc;
+    }
+  }
+  
+  // product list
+  .product-list {
+    width: 75%;
+    padding: 0;
+    @include break-point(tablet) {
+      padding: 5px;
+    }
+  
+    .product-list-row {
+      display: flex;
+      width: 100%;
+      flex-wrap: wrap;
+    }
+  }
+  
+  // product card
+  .product-card {
+    width: 100%;
+    padding: 5px;
+    margin-top: 10px;
+    border-bottom: 1px dashed #d0d0d0;
+    box-shadow: 10px 0 5px -13px #888;
+    @include break-point(tablet) {

```

Comment:
This file probably have 20+ uses of @include break-point where as actual view ports are only 3